docs/overview/trustbloc-did-method.md

```diff
@@ -167,17 +196,37 @@ Most operations require the resolver to process configuration files before findi
 
 - Given a bloc DID: `did:trustbloc:[domain]:[hash]`
 - Check local cache for fresh config files from a previous bootstrapping.
-- If there isn't any cached consortium config that matches the DID's domain, the response depends on the configuration of the resolver - it could abort, or automatically bootstrap trust with the consortium, caching the config.
+- If there isn't any cached consortium config that matches the DID's domain, the response depends on the configuration of the client - it could abort, or automatically bootstrap trust with the consortium, caching the config.
```

Comment:
Should add some explanatory text that after the client aborts, its operator can then decide to examine the validated config and set it trusted. Note: validated means that the client received sufficient matching configurations from different organizations.